pyproject.toml

```diff
@@ -113,11 +111,11 @@ MF_TEST_ADAPTER_TYPE="duckdb"
 MF_SQL_ENGINE_URL="duckdb://"
 # This allows us to use the classes in the `dbt-metricflow` package for tests without installing the package.
 # `dbt-metricflow` can't be installed as it has `metricflow` as a dependency.
-PYTHONPATH="./dbt-metricflow"
+PYTHONPATH="metricflow-semantics:dbt-metricflow"
```

Comment:
Ah, nice, I see, so this should maybe make the install from the last PR editable? Let's give it a try!